app/assets/javascripts/discourse/app/lib/d-popover.js

```diff
@@ -26,15 +26,38 @@ export const hideOnEscapePlugin = {
   },
 };
 
+// legacy, shouldn't be needed with setup
 export function hidePopover(event) {
   if (event?.target?._tippy) {
     showPopover(event);
   }
 }
 
+// legacy, setup() should be used
+export function showPopover(event, options = {}) {
+  const instance = event.target._tippy
+    ? event.target._tippy
+    : setup(event.target, options);
+
+  // hangs on legacy ember
+  if (!isLegacyEmber) {
+    run.begin();
+    instance.popper.addEventListener("transitionend", run.end, {
+      once: true,
+    });
+  }
+
+  if (instance.state.isShown) {
+    instance.hide();
+  } else {
+    instance.show();
+  }
+}
+
+// target is the element trigger the display of the popover
```

Comment:
Am I reading this right? 

```suggestion
// target is the element that triggers the display of the popover
```

Reply:
yes better indeed :+1: